lib/coins.py

```diff
@@ -883,6 +883,22 @@ class Zcash(EquihashMixin, Coin):
     RPC_PORT = 8232
     REORG_LIMIT = 800
 
+class snowgem(EquihashMixin, Coin):
```

Comment:
Please call the class and its NAME SnowGem following the convention of other coin names.  That is also what snowgem.org does.  The rest is good.

Reply:
Done, thanks for fast reply. :)